tutorial/README.md

```diff
@@ -0,0 +1,391 @@
+
+If using a local Prefect server instance instead of Prefect Cloud, start your server by running the following command:
+
+```bash
+prefect server start 
+```
+
+### Step 5: Clone the repository (optional)
+
+You can code from scratch or clone the repository to get the code files for the examples.
+
+```bash
+git clone https://github.com/PrefectHQ/prefect-background-task-examples.git
+```
+
+Move into the directory.
+
+```bash
+cd prefect-background-task-examples
+```
+
+Alright, let's run some tasks!
+
+## Example 1: Run a Prefect task outside of a flow
+
+Add the `@task` decorator to any Python function to define a Prefect task.
+
+Step 1: Create a file named `greeter.py` and save the following code in it, or run the existing file in the [tutorial directory](./tutorial).
+
+```python
+from prefect import task 
+
+@task(log_prints=True)
+def greet(name: str = "Marvin"):
+    print(f"Hello, {name}!")
+
+if __name__ == "__main__":
+    greet()
+```
+
+Step 2: Run the script in the terminal.
+
+```bash
+python greeter.py
+```
+
+You should see the task run in the terminal.
+
+Optional:
+
+You can see the task run in the UI (when the task run page is implemented - coming soon!).
+If you're using a self-hosted Prefect Server instance, you can also see the task runs in the database.
+
+If you want to inspect the SQLite database, use your favorite interface.
+We explain how to use *DB Browser for SQLite* below.
+
+Download it [here](https://sqlitebrowser.org/dl/), if needed. Install it and open it.
+
+Click *Connect*. Then navigate to your SQLite DB file. It will be in the `~/.prefect` directory by default.
+
+Head to the `task_run` table and you should see all your task runs there.
+You can scroll down to see your most recent task runs or filter for them.
+
+Hit the refresh button for updates, if needed.
+
+## Example 2: Start a task server and run tasks in the background
+
+In this example, we'll start a task server and run tasks in the background.  
+
+To run tasks in a separate process or container, you'll need to start a task server, similar to how you would run a Celery worker or an arq worker.
+The task server will continually receive submitted tasks to execute from Prefect's API, execute them, and report the results back to the API.
+You can run a task server by passing tasks into the `prefect.task_server.serve()` method.
+
+Step 1: Define the task and task server in the file `task_server.py`
+
+```python
+from prefect import task
+from prefect.task_server import serve
+
+
+@task
+def my_background_task(name: str):
+    print(f"Hello, {name}!")
+
+
+if __name__ == "__main__":
+    # if on prefect 2.16.4 or older add the following line
+    # from task_server import my_background_task
+
+    serve(my_background_task)
+```
+
+Step 2: Start the task server by running the script in the terminal.
+
+```bash
+python task_server.py
```

Comment:
what do you think about calling this `tasks.py` instead of `task_server.py`?

something strikes me weird about importing tasks to submit from `task_server`